requirements/edx/base.txt

```diff
@@ -58,7 +58,7 @@ beautifulsoup4==4.11.1
     # via pynliner
 billiard==3.6.4.0
     # via celery
-bleach[css]==5.0.1
```

Comment:
I'm not sure why these lines have changed -- when I do a `make compile-requirements`, they're restored. Usually these kinds of differences come down to a Linux vs. Mac thing where some packages declare that they only need certain dependencies on certain OSes, but I don't _think_ that's what's going on here. :thinking:

Reply:
Good catch, it turned out this was important and so I made https://github.com/openedx/edx-platform/pull/31367 to fix this so that hopefully it will be more consistent in the future.